chart/values.yaml

```diff
@@ -300,6 +300,16 @@ kube-prometheus-stack:
           severity: 'warning'
         equal: ['alertname']
 
+# Values for Openshift clusters
+openshift:
+  # Set to true if you use openshift cluster
+  enabled: false
+  # If null, the helmchart creates a new scc
+  existingScc: null
```

Comment:
For me, it would make more sense for me when you name it like `existingSecurityContextConstraints` to be consistent with below ones. You do not use shortcut for `kubePrometheusServiceAccounts` etc.